src/api/app/assets/stylesheets/webui2/monitor.scss

```diff
@@ -0,0 +1,5 @@
+#building,
+#events,
+#jobs {
```

Comment:
as this moved not into the global namespace of css, I would recommend to pick a little longer ids (or scope them)

Reply:
@coolo better now?